devops/helm/crunchy-postgres/Makefile

```diff
@@ -18,7 +18,7 @@ helm-dep:
 .PHONY: install
 install: helm-dep
 install:
-	@helm install $(call arguments) --set restore.enabled=false -f values.yaml
```

Comment:
Can you clarify why `-f values.yaml` is changed?
The PR description mentioned the size change and did not explain this additional change.

Reply:
Updated the PR description.